src/Base/FileInfo.cpp

```diff
@@ -456,52 +456,44 @@ unsigned int FileInfo::size() const
     return 0;
 }
 
-TimeInfo FileInfo::lastModified() const
+time_t FileInfo::lastModified() const
 {
-    TimeInfo ti = TimeInfo::null();
+    time_t res = (time_t)(-1);
     if (exists()) {
-
 #if defined(FC_OS_WIN32)
         std::wstring wstr = toStdWString();
         struct _stat st;
         if (_wstat(wstr.c_str(), &st) == 0) {
-            ti.setTime_t(st.st_mtime);
+            res = st.st_mtime;
         }
-
 #elif defined(FC_OS_LINUX) || defined(FC_OS_CYGWIN) || defined(FC_OS_MACOSX) || defined(FC_OS_BSD)
-        struct stat st
-        {
-        };
+        struct stat st;
         if (stat(FileName.c_str(), &st) == 0) {
-            ti.setTime_t(st.st_mtime);
+            res = st.st_mtime;
         }
 #endif
     }
-    return ti;
+    return res;
 }
 
-TimeInfo FileInfo::lastRead() const
+time_t FileInfo::lastRead() const
 {
-    TimeInfo ti = TimeInfo::null();
+    time_t res = (time_t)(-1);

```

Comment:
Is this idiomatic? `time_t`'s type isn't defined by the standard.

Reply:
I just needed something to indicate error. As time(2) man page says '_On error, ((time_t) -1) is returned, and errno is set to indicate the error._' I will probably implement `TimeInfo` class again based on `std::chrono` once `FileInfo` gets reimplemented using `std::filesystem`.